site/en/blog/private-network-access-preflight/index.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: 'layouts/blog-post.njk'
+title: "Private Network Access: introducing preflights"
+authors:
+  - titouan
+  - agektmr
+description: Chrome is deprecating access to private network endpoints from non-secure public websites as part of the Private Network Access specification. Read on for recommended actions.
+date: 2022-01-06
+hero: image/VbsHyyQopiec0718rMq2kTE1hke2/iqanYAE91Ab6BsgwhBjq.jpg
+alt: An airplane in the sky
+tags:
+  - chrome-98
+  - privacy
+---
+
+## Introduction
+
+Chrome is deprecating direct access to private network endpoints from public
+websites as part of the
+[Private Network Access](https://wicg.github.io/private-network-access/) (PNA)
+specification.
+
+Chrome will start sending a CORS preflight request ahead of any [private
+network request](#what-is-private-network-access) for a subresource, which asks
+for explicit permission from the target server. This preflight request will
+carry a new header, `Access-Control-Request-Private-Network: true`, and the
+response to it must carry a corresponding header,
+`Access-Control-Allow-Private-Network: true`.
+
+The aim is to protect users from cross-site request forgery (CSRF) attacks
```

Comment:
```suggestion
The aim is to protect users from [cross-site request forgery (CSRF) attacks](https://portswigger.net/web-security/csrf)
```

Add a link to the attack details. This link is also used at https://web.dev/security-headers/